src/components/User-Landing/User-Landing.js

```diff
@@ -1,135 +1,99 @@
-// Imports
 import React, { useState, useEffect } from 'react';
-import { Link } from 'react-router-dom';
-import PropTypes from "prop-types";
-
-// Component Imports
+import { Link, Redirect } from 'react-router-dom';
 import UpdateUserInfoForm from '../Update-Form/Update-Form';
+import { useQuery } from '@apollo/client'
+import GET_USER from '../../GraphQL/queries'
+
+const UserLandingPage = () => {
+
+  const { error, loading, data } = useQuery(GET_USER, {variables: {id: 2}})
+
+
+  const [user, setUser] = useState({})
```

Comment:
I usually set empty objects as null, not sure if it matters but wanted to point it out